sdk-workflows/src/main/java/io/dapr/workflows/client/DaprWorkflowClient.java

```diff
@@ -42,4 +43,30 @@ public DaprWorkflowClient(DurableTaskClient innerClient) throws IllegalArgumentE
   public String scheduleNewWorkflow(String workflowName) {
     return this.innerClient.scheduleNewOrchestrationInstance(workflowName);
   }
+
+  //TODO: Overloaded scheduleNewWorkflow implementations
+
+  public OrchestrationMetadata getWorkflowMetadata(String workflowName) {
+    return this.innerClient.getInstanceMetadata(workflowName, false); //TODO: is false what we want?
+  }
+
+  /**
+   * Terminates the workflow associated with the provided instance id.
+   *
+   * @param workflowName name of workflow to terminate
+   */
+  public void terminateWorkflow(String workflowName) {
```

Comment:
We should expose the same arguments as the durableTaskClient here and pass it along

```suggestion
  public void terminateWorkflow(String workflowId, @Nullable Object output) {
```